ci-operator/step-registry/openshift/upgrade/qe/test/cert-rotation/openshift-upgrade-qe-test-cert-rotation-commands.sh

```diff
@@ -3,11 +3,19 @@ set -xeuo pipefail
 
 function download_oc(){
     local tmp_bin_path='/tmp/oc-bin/'
-    mkdir -p "$tmp_bin_path"
-    curl -sSL --retry 3 --retry-delay 5 https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest-4.10/openshift-client-linux.tar.gz | tar xvzf - -C "${tmp_bin_path}" oc
-    export PATH=${tmp_bin_path}:$PATH
-    which oc
-    oc version --client
+    # 4.6 using latest 4.10 oc client, hits the bug https://issues.redhat.com/browse/OCPBUGS-16009, it has been fixed on 4.14.
+    # In order for the test to continue, we have to use 4.14 nightly oc client temporarily, once 4.10 is available, will remove the code.
+    if [[ ${major_version} -eq 4 && ${minor_version} -eq 6 ]];then
```

Comment:
A note, I'm not sure whether the `major|minor_version` is a number or a string. If it's already validated that it's a number, then please ignore this comment.

Reply:
I tested, it is a number 
```
$ ocp_version=$(oc get -o jsonpath='{.status.desired.version}' clusterversion version)
$ major_version=$(echo ${ocp_version} | cut -d '.' -f1)
$ minor_version=$(echo ${ocp_version} | cut -d '.' -f2)
$ echo $major_version $minor_version
4 9

```